doc/operations/data_persistence.rst

```diff
@@ -0,0 +1,26 @@
+Data Persistence
+=====================
+
+When a new node has joined the network or when a failed node needs to be recovered, latest committed snapshot file can be copied to the node before it is started. It will then automatically resume from the latest snapshot file (see :ref:`operations/ledger_snapshot:Join/Recover From Snapshot`).
+
+The new/recovered node may also need to have access to all ``.committed`` ledger files in some cases, for example, if the node needs to serve historical queries. It is therefore safe to back up all the ``.committed`` ledger and snapshot files. It is recommended to have two separate directories on each node - one being a read-write directory where *all* the ledger and snapshot files reside and another shared read-only directory where *only* the ``.committed`` ledger and snapshot files reside.
+
+.. mermaid::
+
+    graph TD;
+    subgraph ccf network;
+    A["Node 0 (Primary)<br> [Read-Write directory]"];
+    B["Node 1<br> [Read-Write directory]"];
+    C["Node 2<br> [Read-Write directory]"];
+    D["Shared Mount<br>[Read-Only directory]"]
+    style D fill:#bbf,stroke:#f66,stroke-width:2px,color:#fff,stroke-dasharray: 5 5
+    A-->D
+    A-. copy files .-> D
+    B-->D
+    C-->D
+    end;
+
+The read-only directory could be a shared mounted directory which is accessible to all the nodes in the network. The shared read-only ledger and snapshot directories can be specified via the ``ledger.read_only_directories`` and ``snapshot.read_only_directories`` configuration options respectively.
```

Comment:
```suggestion
The read-only directory could be a shared mounted directory which is accessible to all the nodes in the network. The shared read-only ledger and snapshot directories can be specified via the ``ledger.read_only_directories`` and ``snapshots.read_only_directory`` configuration options respectively.
```

Looking at the current config, we only support a single read-only snapshot directory.